prow/scripts/cluster-integration/kyma-gke-central.sh

```diff
@@ -118,7 +118,7 @@ ERROR_LOGGING_GUARD="true"
 
 log::info "Authenticate"
 gcloud::authenticate
-INSTALL_DIR=$(mktemp -d) install::kyma_cli
+INSTALL_DIR=$(mktemp -d) kyma::install_cli
```

Comment:
We don't need temporary directory creation anymore. You can safely remove `INSTALL_DIR=$(mktemp -d)`